src/domains/receipts-common/env/weu-dev/terraform.tfvars

```diff
@@ -0,0 +1,57 @@
+prefix         = "pagopa"
+env_short      = "d"
+env            = "dev"
+domain         = "receipts"
+location       = "westeurope"
+location_short = "weu"
+instance       = "dev"
+
+tags = {
+  CreatedBy   = "Terraform"
+  Environment = "Prod"
+  Owner       = "pagoPA"
+  Source      = "https://github.com/pagopa/pagopa-infra/tree/main/src/receipts"
+  CostCenter  = "TS310 - PAGAMENTI & SERVIZI"
+}
+
+### External resources
+
+monitor_resource_group_name                 = "pagopa-d-monitor-rg"
+log_analytics_workspace_name                = "pagopa-d-law"
+log_analytics_workspace_resource_group_name = "pagopa-d-monitor-rg"
+application_insights_name                   = "pagopa-d-appinsights"
+
+### Aks
+
+ingress_load_balancer_ip = "10.1.100.250"
+
+external_domain          = "pagopa.it"
+dns_zone_internal_prefix = "internal.dev.platform"
+
+# CosmosDB Receipts Datastore
+receipts_datastore_cosmos_db_params = {
+  kind         = "GlobalDocumentDB"
+  capabilities = ["EnableServerless"]
+  offer_type   = "Standard"
+  consistency_policy = {
+    consistency_level       = "Strong"
+    max_interval_in_seconds = 300
+    max_staleness_prefix    = 100000
+  }
+  server_version                   = "4.0"
+  main_geo_location_zone_redundant = false
+  enable_free_tier                 = false
+
+  additional_geo_locations          = []
+  private_endpoint_enabled          = false
+  public_network_access_enabled     = true
+  is_virtual_network_filter_enabled = false
+
+  backup_continuous_enabled = false
+
+  container_default_ttl = 2629800 # 1 month in second
+}
+
+cidr_subnet_receipts_datastore_cosmosdb = ["10.1.156.0/24"]
```

Comment:
to verify the right cidr should see [here](https://portal.azure.com/?l=en.en-us#@pagopait.onmicrosoft.com/resource/subscriptions/bbe47ad4-08b3-4925-94c5-1278e5819b86/resourceGroups/pagopa-d-vnet-rg/providers/Microsoft.Network/virtualNetworks/pagopa-d-vnet/subnets) 
try with `10.1.170.0/24`